source/agora/consensus/protocol/Nominator.d

```diff
@@ -905,9 +898,16 @@ extern(D):
 
         if (auto fail_reason = this.ledger.validateConsensusData(data, this.initial_missing_validators))
         {
-            log.error("validateValue(): Validation failed: {}. Data: {}",
-                fail_reason, data.prettify);
-            return ValidationLevel.kInvalidValue;
+            if (fail_reason == this.ledger.InvalidConsensusDataReason.NotInPool)
+            {
+                log.trace("validateValue(): This node can not yet fully validate this value: {}", fail_reason);
```

Comment:
We might want to log the data as well, similar to the else case

Reply:
The difference is that we know that we do not have all txs, and we log the number missing in `getValidTXSet`. I'll add it as a debug log though.